packages/fury-adapter-oas3-parser/lib/parser/oas/parseParameterObject.js

```diff
@@ -27,9 +27,9 @@ const isValidInValue = R.anyPass([
   hasValue('cookie'),
 ]);
 
-
+const unreservedCharacterRegex = /^[A-z0-9\\.\\_\\~\\-]+$/;
```

Comment:
I've moved out the creation of the regex expression outside of the function for performance reasons. We only have to compile the regex expression once when the parser is loaded instead of for each parameter.